
bgo#925101 for some unknown reason some users are having issues with pgo crashing. Forcing software
rendering to be used while pgo'ing doesn't have an impact in performance, as hwaccel isn't used
during pgo even when it works. The issue could be caused by wrongly/automatically
configured/detected gpu access anyway. The issue seems related to mesa, but hard to pinpoint.

diff '--color=auto' -Naur a/testing/profiles/profileserver/user.js b/testing/profiles/profileserver/user.js
--- a/testing/profiles/profileserver/user.js	2024-07-11 10:37:53.889115118 +0300
+++ b/testing/profiles/profileserver/user.js	2024-07-11 10:40:23.086996283 +0300
@@ -11,3 +11,6 @@
 user_pref("extensions.webextensions.warnings-as-errors", false);
 // Turn off update
 user_pref("app.update.disabledForTesting", true);
+// bgo#925101 mesa issues with pgo, force software rendering during pgo
+user_pref("gfx.webrender.software", true);
+
